cirq/contrib/two_qubit_gates/example.py

```diff
@@ -0,0 +1,61 @@
+from time import time
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+# pylint: disable=line-too-long
```

Comment:
Instead of disabling the linter, I feel like we should probably just let yapf format this. It will probably do something like:
from cirq.contrib.two_qubit_gates.math_utils import (
  random_two_qubit_unitaries_and_kak_vecs,
  unitary_entanglement_fidelity,
)

Reply:
yapf doesn't seem to fix this...

Reply:
Put parens around the import list and then yapf will fix it. Don't disable the line too long check.

Reply:
Done